megamek/src/megamek/common/IAero.java

```diff
@@ -787,5 +787,27 @@ default void useFuel(int fuelUsed) {
      */
     default void updateSensorOptions() {
     }
+    
+    // Auto Ejection Methods
+    
+    boolean isAutoEject();
+    
+    void setAutoEject(boolean autoEject);
+    
+    boolean isCondEjectAmmo();
+   
+    void setCondEjectAmmo(boolean condEjectAmmo);
+    
+    boolean isCondEjectFuel();
+    
+    void setCondEjectFuel(boolean condEjectFuel);
```

Comment:
The fuel condition is only used for Aero fuel tank critical hits, and never used where it could also apply to a LAM.

Reply:
Fixed.